components/event-receiver/event-input-adapters/org.wso2.carbon.event.input.adapter.mqtt/pom.xml

```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+~ Copyright (c) 2015, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+~
+~ Licensed under the Apache License, Version 2.0 (the "License"); you may not
+~ use this file except in compliance with the License. You may obtain a copy
+~ of the License at
+~
+~ http://www.apache.org/licenses/LICENSE-2.0
+~
+~ Unless required by applicable law or agreed to in writing, software distributed
+~ under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+~ CONDITIONS OF ANY KIND, either express or implied.  See the License for the
+~ specific language governing permissions and limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>event-input-adapters</artifactId>
+        <groupId>org.wso2.carbon.event-processing</groupId>
+        <version>2.0.4-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>org.wso2.carbon.event.input.adapter.mqtt</artifactId>
+    <packaging>bundle</packaging>
+    <version>2.0.4-SNAPSHOT</version>
```

Comment:
@keizer619 Any reason for repeating the version here as well as under parent element?